### DISCUSSION TITLE
🔗 Floating Buttons: Custom Icons (Duplicate of #28741)

### DISCUSSION THREAD

**Discussion by sflwa** [elementor/elementor#28779] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The new feature of floating icons is create - however it is limited by the "platform" type - If you want to do anything custom or multiple URLs they all have a global

### Describe the solution you'd like.

Give the ability to have a "custom" which allows you to put in a custom URL / link as well as select the icon as this could be used for links to PDFs / social media / other systems

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by sflwa** [comment]
I just found a similar post so I'm closing this

**Comment by nicholaszein** [comment]
Hello there!

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #28741

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
